.gitignore

```diff
@@ -0,0 +1,61 @@
+# Logs
```

Comment:
Should the package-lock.json's be added to the gitignore? File locking isn't supposed to be committed, but is this an aspect of JS that is different?

Reply:
`package-lock.json` just locks package dependencies, NPM recommends we commit it to source control. What exactly it does I'm still not sure, but you can read up about it more here:
https://docs.npmjs.com/files/package-lock.json